ioctl/newcmd/account/accountdelete.go

```diff
@@ -0,0 +1,143 @@
+	}
+	failToWriteToConfigFile = map[config.Language]string{
+		config.English: "Failed to write to config file %s.",
+		config.Chinese: "写入配置文件 %s 失败",
+	}
+	resultSuccess = map[config.Language]string{
+		config.English: "Account #%s has been deleted.",
+		config.Chinese: "账户 #%s 已被删除",
+	}
+	failToFindAccount = map[config.Language]string{
+		config.English: "account #%s not found",
+		config.Chinese: "账户 #%s 未找到",
+	}
+)
+
+// NewAccountDelete represents the account delete command
+func NewAccountDelete(c ioctl.Client) *cobra.Command {
+
+	use, _ := c.SelectTranslation(deleteUses)
+	short, _ := c.SelectTranslation(deleteShorts)
+	failToGetAddress, _ := c.SelectTranslation(failToGetAddress)
+	failToConvertStringIntoAddress, _ := c.SelectTranslation(failToConvertStringIntoAddress)
+	infoWarn, _ := c.SelectTranslation(infoWarn)
+	failToRemoveKeystoreFile, _ := c.SelectTranslation(failToRemoveKeystoreFile)
+	failToWriteToConfigFile, _ := c.SelectTranslation(failToWriteToConfigFile)
+	resultSuccess, _ := c.SelectTranslation(resultSuccess)
+	failToFindAccount, _ := c.SelectTranslation(failToFindAccount)
+
+	ad := &cobra.Command{
+		Use:   use,
+		Short: short,
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			arg := ""
+			if len(args) == 1 {
+				arg = args[0]
+			}
+			addr, err := c.GetAddress(arg)
+			if err != nil {
+				return output.NewError(output.AddressError, failToGetAddress, err)
+			}
+			account, err := address.FromString(addr)
+			if err != nil {
+				return output.NewError(output.ConvertError, fmt.Sprintf(failToConvertStringIntoAddress),
+					nil)
+			}
+			ks := c.NewKeyStore(config.ReadConfig.Wallet,
+				keystore.StandardScryptN, keystore.StandardScryptP)
+			//ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+			//keystore.StandardScryptN, keystore.StandardScryptP)
+			for _, v := range ks.Accounts() {
+				if bytes.Equal(account.Bytes(), v.Address.Bytes()) {
+					var confirm string
+					info := fmt.Sprintf(infoWarn)
+					message := output.ConfirmationMessage{Info: info, Options: []string{"yes"}}
+					fmt.Println(message.String())
+					fmt.Scanf("%s", &confirm)
+					if !strings.EqualFold(confirm, "yes") {
+						output.PrintResult("quit")
+						return nil
+					}
+
+					if err := os.Remove(v.URL.Path); err != nil {
+						return output.NewError(output.WriteFileError, failToRemoveKeystoreFile, err)
+					}
+
+					aliases := alias.GetAliasMap()
+					delete(config.ReadConfig.Aliases, aliases[addr])
+					out, err := yaml.Marshal(&config.ReadConfig)
+					if err != nil {
+						return output.NewError(output.SerializationError, "", err)
+					}
+					if err := ioutil.WriteFile(config.DefaultConfigFile, out, 0600); err != nil {
+						return output.NewError(output.WriteFileError,
+							fmt.Sprintf(failToWriteToConfigFile, config.DefaultConfigFile), err)
+					}
+					output.PrintResult(fmt.Sprintf(resultSuccess, addr))
```

Comment:
we shouldn't referring to package "alias", which was defined in https://github.com/iotexproject/iotex-core/tree/master/ioctl/cmd/alias. Eventually, we need to delete everything in ioctl/cmd